gradio/mcp.py

```diff
@@ -262,7 +262,7 @@ async def handle_sse(request):
                 routes=[
                     Route(
                         "/schema",
-                        endpoint=self.get_complete_schema,  # Not required for MCP but useful for debugging
+                        endpoint=self.get_complete_schema,  # Not required for MCP but used by the Hugging Face MCP server to get the schema for MCP Spaces without needing to establish an SSE connection
```

Comment:
Was chatting with @evalstate who mentioned that the HF MCP server uses the schema, so I figured it's good to note that in the codebase so we don't accidentally make a breaking change.